scripts/issue/620/resolve_badger_deficit.py

```diff
@@ -0,0 +1,17 @@
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main():
+    vault = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+    badger = interface.ERC20(registry.eth.treasury_tokens.BADGER, owner=vault.account)
+
+    vault.take_snapshot([badger])
+
+    # current deficit is ~198k badger + 19k badger which will be emitted between week 27 - 31
+    # https://github.com/Badger-Finance/badger-multisig/issues/620#issuecomment-1177909957
+    badger.transfer(registry.eth.drippers.tree_2022_q3, 223_000e18)
```

Comment:
i think you failed to take the pending badger from the dripper into account: https://etherscan.io/tx/0xe4c58295b26e807fd9d75091a43354338303229e98403701d55083acacc36518
```suggestion
    # current deficit is ~198k badger + 19k badger which will be emitted between week 27 - 31
    # https://github.com/Badger-Finance/badger-multisig/issues/620#issuecomment-1177909957
    badger.transfer(registry.eth.drippers.tree_2022_q3, 190_000e18)
```

Reply:
great point. i consider that amount now via latest commit: [8091b30](https://github.com/Badger-Finance/badger-multisig/pull/621/commits/8091b3093ba3e6b707c7068912e1e813a2f28342)